lxd/devlxd.go

```diff
@@ -379,14 +379,15 @@ type ConnPidMapper struct {
 func (m *ConnPidMapper) ConnStateHandler(conn net.Conn, state http.ConnState) {
 	unixConn, _ := conn.(*net.UnixConn)
 	if unixConn == nil {
-		logger.Error("Invalid type for devlxd net conn", logger.Ctx{"conn_type": fmt.Sprintf("%T", conn)})
+		logger.Error("Invalid type for devlxd connection", logger.Ctx{"conn_type": fmt.Sprintf("%T", conn)})
+		return
```

Comment:
Is this a bug fix, should it be its own commit?

Reply:
Apologies, this should have been in the previous commit. I realised that we need to return here if the `net.Conn` is not a `*net.UnixConn` or the function will panic.